cpp/oneapi/dal/backend/primitives/reduction.hpp

```diff
@@ -15,3 +15,5 @@
 *******************************************************************************/
 
 #pragma once
+
+#include "oneapi/dal/backend/primitives/reduction/reduction.hpp"
```

Comment:
This file does not exist

Reply:
For some reason `reduction/reduction.hpp` was renamed to `reduction/reduction_dpc.hpp`. I suggest to leave the old name. Let's not distinguish headers for C++ and DPC++.